README.md

```diff
@@ -28,109 +37,129 @@ Where site-specific values for the representative clutter height are not availab
+The [Study Group Clutter Excel Workbook](https://www.itu.int/en/ITU-R/study-groups/rsg3/ionotropospheric/Clutter%20and%20BEL%20workbook_V2.xlsx)
+contains an extensive set of validation example values.
 
 ## Notes on Code Style ##
 
- * In general, variables follow the naming convention in which a single underscore denotes a subscript (pseudo-LaTeX format), where a double underscore is followed by the units, i.e. h_1__meter.
- * Variables are named to match their corresponding mathematical variables in the underlying Recommendation text.
- * Wherever possible, equation numbers are provided.  It is assumed that a user reviewing this source code would have a copy of the Recommendation's text available as a primary reference.
+* In general, variables follow the naming convention in which a single underscore
+denotes a subscript (pseudo-LaTeX format), where a double underscore is followed
+by the units, i.e. `h_1__meter`.
+* Variables are named to match their corresponding mathematical variables in the
+underlying Recommendation text.
+* Wherever possible, equation numbers are provided. It is assumed that a user
+reviewing this source code would have a copy of the Recommendation's text available
+as a primary reference.
 
 ## Configure and Build ##
 
-### C++ Software
+### C++ Software ###
 
-The software is designed to be built into a DLL (or corresponding library for non-Windows systems).  The source code can be built for any OS that supports the standard C++ libraries.  A Visual Studio 2019 project file is provided for Windows users to support the build process and configuration.
+The software is designed to be built into a DLL (or corresponding library for
+non-Windows systems). The source code can be built for any OS that supports the
+standard C++ libraries. A Visual Studio 2019 project file is provided for Windows
+users to support the build process and configuration.
 
-### C#/.NET Wrapper Software
+### C#/.NET Wrapper Software ###
 
-The .NET support of P.2108 consists of a simple pass-through wrapper around the native DLL.  It is compiled to target .NET Framework 4.7.2.  Distribution and updates are provided through the published [NuGet package](https://github.com/NTIA/p2108/packages).
+The .NET support of P.2108 consists of a simple pass-through wrapper around the
+native DLL. It is compiled to target .NET Framework 4.8.1. Distribution and updates
+are provided through the published [NuGet package](https://github.com/NTIA/p2108/packages).
 
 ## References ##
 
- * [Recommendation ITU-R P.2108](https://www.itu.int/rec/R-REC-P.2108-0-201706-I/en)
- * [Report ITU-R P.2402](https://www.itu.int/pub/R-REP-P.2402)
+* [Recommendation ITU-R P.2108](https://www.itu.int/rec/R-REC-P.2108/en)
+* [Report ITU-R P.2402](https://www.itu.int/pub/R-REP-P.2402)
 
 ## Contact ##
 
-For questions, contact Billy Kozma, (303) 497-6082, wkozma@ntia.gov
+For questions, contact Billy Kozma, (303) 497-6082, <wkozma@ntia.gov>
```

Comment:
I've been stripping out phone numbers from repos.  Both cause this number doesnt exist anymore (landline) and cause of spam.  So just use name/email for contact info

Reply:
Removed in afb17885ba493bc5cd1e4af837cdf57bf7449114